repl/scala-2.10/src/main/scala/org/apache/spark/repl/SparkILoopInit.scala

```diff
@@ -127,7 +127,17 @@ private[repl] trait SparkILoopInit {
            _sc
          }
         """)
+      command("""
+         @transient val sqlContext = {
+           val _sqlContext = org.apache.spark.repl.Main.interp.createSQLContext()
+           println("SQLContext available as sqlContext.")
+           _sqlContext
+         }
+        """)
       command("import org.apache.spark.SparkContext._")
+      command("import sqlContext._")
```

Comment:
let's not use wildcard import of sqlContext. It adds too much stuff. 

Once this goes in, maybe you can import sqlContext.implicits._
https://github.com/apache/spark/pull/4386